Makefile

```diff
@@ -0,0 +1,23 @@
+.PHONY: run-server run-client db-start migrate-up migrate-down setup
+
+run-server:
+	go run *.go
```

Comment:
Just curious - why `go run *.go` instead of `go run main.go`? Easier to maintain in case we change the file name? nit - maybe slightly more obvious to run an explicit file.

Reply:
`go run` requires all the files containing source code to be provided as arguments. A better alternative would probably be to use `go build` instead, which takes in package names as arguments.